compiler/src/main/kotlin/edu/cornell/cs/apl/viaduct/codegeneration/Utility.kt

```diff
@@ -87,3 +88,13 @@ fun receiveReplicated(
 
     return receiveExpression.build()
 }
+
+fun getRole(protocol: Protocol, host: Host): CodeBlock =
```

Comment:
This is ABY specific, not a general purpose utility function. Should be moved into the `ABYCodeGenerator` class.